### ISSUE TITLE
spanish docs

### ISSUE THREAD

**Issue by baileyb0t** [HRDAG/training-docs#26]
We are waiting to meet with someone who specializes in localization to discuss the right way to structure/organize the spanish materials (separate repo? fork?), but @sierrawells has already kicked this off by translating some of the `onboarding/` docs. Thank you, Sierra! 🤩

**Comment by sierrawells** [comment]
In `training-docs/README.md`, there are a couple of links to the issue section of the [HRDAG-training repo](https://github.com/HRDAG/HRDAG-training). It doesn't seem like this repo exists anymore (or at least I get a message that this page no longer exists). Is there a different repo I should link to for resources for troubleshooting when testing out the tools described in the training docs?

**Comment by baileyb0t** [comment]
Hmm, I was able to click on all the links in `training-docs/README.md`, so it's possible it's an issue of access/permissions. Many of our repos are private and/or restricted by group, which is something we might want to change or address for at least the ones linked in training. Alternatively, if the links are meant to be to the training-docs repo itself which is what it sounds like, it might be a bad link from when the repo was originally created under my user.

You mentioned the links refer to the issue section but I'm not seeing which ones refer to issues, can you share which links or prompts don't work for you?

**Comment by baileyb0t** [comment]
@sierrawells I think I figured it out, you were looking at `training-docs/onboarding/README.md`, which is why I didn't find the mention of issues or bad links in `training-docs/README.md`. I found the links you were talking about and updated both. Lmk if they still don't work for you!

**Comment by sierrawells** [comment]
@baileyb0t yes, you are right, sorry about that! I was talking about `training-docs/onboarding/README.md`. All the links now work for me. Thanks and sorry about the confusion! I will update the links in the Spanish onboarding directory accordingly.
